FactoryWeather/App/VIP Home/Views/HeaderView.swift

```diff
@@ -24,7 +24,7 @@ class HeaderView: UIView {
 }
 
 extension HeaderView {
```

Comment:
The `HeaderView` should have a `ViewModel` typealias that contains only the data relevant to the `HeaderView` and the `updateProperties` modified to work with the `HeaderView.ViewModel`.